Sources/Interaction/Manipulators/CameraManipulator/index.js

```diff
@@ -33,38 +26,21 @@ function vtkCameraManipulator(publicAPI, model) {
 // ----------------------------------------------------------------------------
 
 const DEFAULT_VALUES = {
-  manipulatorName: 'goldschlager',
-
-  button: 1,
-  shift: false,
-  control: false,
-  alt: false,
-
   center: [0, 0, 0],
   rotationFactor: 1,
   displayCenter: [0, 0],
-
-  pinch: false,
 };
 
 // ----------------------------------------------------------------------------
 
 export function extend(publicAPI, model, initialValues = {}) {
   Object.assign(model, DEFAULT_VALUES, initialValues);
 
-  // Object methods
-  macro.obj(publicAPI, model);
+  // Inheritance
+  vtkMouseManipulator.extend(publicAPI, model, initialValues);
```

Comment:
@jourdain can you confirm this part is right?

Reply:
yes it is